Challenge1/src/agent.spec.ts

```diff
@@ -1,74 +1,108 @@
-// import {
-//   FindingType,
-//   FindingSeverity,
-//   Finding,
-//   HandleTransaction,
-//   createTransactionEvent,
-//   ethers,
-// } from "forta-agent";
-// import agent, {
-//   ERC20_TRANSFER_EVENT,
-//   TETHER_ADDRESS,
-//   TETHER_DECIMALS,
-// } from "./agent";
-
-// describe("high tether transfer agent", () => {
-//   let handleTransaction: HandleTransaction;
-//   const mockTxEvent = createTransactionEvent({} as any);
-
-//   beforeAll(() => {
-//     handleTransaction = agent.handleTransaction;
-//   });
-
-//   describe("handleTransaction", () => {
-//     it("returns empty findings if there are no Tether transfers", async () => {
-//       mockTxEvent.filterLog = jest.fn().mockReturnValue([]);
-
-//       const findings = await handleTransaction(mockTxEvent);
-
-//       expect(findings).toStrictEqual([]);
-//       expect(mockTxEvent.filterLog).toHaveBeenCalledTimes(1);
-//       expect(mockTxEvent.filterLog).toHaveBeenCalledWith(
-//         ERC20_TRANSFER_EVENT,
-//         TETHER_ADDRESS
-//       );
-//     });
-
-//     it("returns a finding if there is a Tether transfer over 10,000", async () => {
-//       const mockTetherTransferEvent = {
-//         args: {
-//           from: "0xabc",
-//           to: "0xdef",
-//           value: ethers.BigNumber.from("20000000000"), //20k with 6 decimals
-//         },
-//       };
-//       mockTxEvent.filterLog = jest
-//         .fn()
-//         .mockReturnValue([mockTetherTransferEvent]);
-
-//       const findings = await handleTransaction(mockTxEvent);
-
-//       const normalizedValue = mockTetherTransferEvent.args.value.div(
-//         10 ** TETHER_DECIMALS
-//       );
-//       expect(findings).toStrictEqual([
-//         Finding.fromObject({
-//           name: "High Tether Transfer",
-//           description: `High amount of USDT transferred: ${normalizedValue}`,
-//           alertId: "FORTA-1",
-//           severity: FindingSeverity.Low,
-//           type: FindingType.Info,
-//           metadata: {
-//             to: mockTetherTransferEvent.args.to,
-//             from: mockTetherTransferEvent.args.from,
-//           },
-//         }),
-//       ]);
-//       expect(mockTxEvent.filterLog).toHaveBeenCalledTimes(1);
-//       expect(mockTxEvent.filterLog).toHaveBeenCalledWith(
-//         ERC20_TRANSFER_EVENT,
-//         TETHER_ADDRESS
-//       );
-//     });
-//   });
-// });
+import { Finding, HandleTransaction, FindingSeverity, FindingType, TransactionEvent } from "forta-agent";
+import agent from "./agent";
+
+let handleTransaction: HandleTransaction;
+const mockNetherMindAddress = "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8";
+const mockFortaRegistryAddress = "0x61447385B019187daa48e91c55c02AF1F1f3F863";
+
+beforeAll(() => {
+  handleTransaction = agent.handleTransaction;
+});
+
+describe("returns empty findings if transaction is not from Nethermind address", () => {
```

Comment:
We usually have one `describe()` block that has all of the different test cases inside of it. So not necessary to have several `describe()` blocks. [Here](https://github.com/NethermindEth/Forta-Agents/blob/main/DYDX-Bots/Large-USDC-Staking-Withdrawals/src/agent.spec.ts#L27) is a quick example of what I mean.